tslint.json

```diff
@@ -17,6 +17,10 @@
     "no-console": false,
     "no-import-un-mocked": true,
     "no-namespace": [true, "allow-declarations"],
+    "no-query-renderer-import": {
+      "severity": "warning",
+      "options": ["blah"]
+    },
```

Comment:
You shouldn't need all this. You can just do `"no-query-renderer-import": true,`

Reply:
Yea, this was just me faffing with specifying it differently to see what was up with that 'missing severity' error.